syntax = "proto3";
service KassaService {
 rpc CreatePayment(CreatePaymentRequest) returns (PaymentResponse);
}
message CreatePaymentRequest {
 string payment_sum = 1;
 string email = 2;
}
message PaymentResponse {
 string payment_id = 1;
 string payment_sum = 2;
 string email = 3;
 string receipt_url = 4;
}
